src/main/java/com/powsybl/openloadflow/network/LfNetworkParameters.java

```diff
@@ -45,8 +45,6 @@ public class LfNetworkParameters {
 
     private double plausibleActivePowerLimit = PLAUSIBLE_ACTIVE_POWER_LIMIT_DEFAULT_VALUE;
 
-    private boolean addRatioToLinesWithDifferentNominalVoltageAtBothEnds = false;
```

Comment:
@geofjamg we have an incoherence here that can be responsible of the TUs that are failing.